runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -121,9 +158,16 @@ private void restoreTable(String fileName, UUID streamId) throws IOException {
                 numEntries += smrEntries.size();
             }
             txn.commit();
-            log.info("completed restore of table {} with {} numEntries", streamId, numEntries);
+
+            long elapsedTime = System.currentTimeMillis() - startTime;
+            restoreTimeDistributionSummary.ifPresent(summary -> summary.record(elapsedTime));
```

Comment:
Record runnable with timer

Reply:
After discussion, I think using Metrics to record backup/restore time per table does not give more insight comparing to using simple log messages. Backup/restore is one-time action and does not fit the aggregation and periodic report fashion of Metrics. However, the corfu runtime Metrics will still be used to monitor the low level statistics such as read/write latency.